src/caked/Transforms/utils.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+import math
+
+import numpy as np
+from ccpem_utils.map.parse_mrcmapobj import MapObjHandle
+
+
+def pad_map_grid_sample(
```

Comment:
Are these different from functions in ccpem-utils?

Reply:
No it isn't, the new code is implemented but this was an artifact from inbetween, I'll remove it